src/kex/kex.c

```diff
@@ -2,8 +2,6 @@
 
 #include <oqs/kex.h>
```

Comment:
We should remove all kex related files at this point (if ntru is the last one)

Reply:
I'll remove that in issue: https://github.com/open-quantum-safe/liboqs/issues/372